Options.lua

```diff
@@ -4176,6 +4176,18 @@ self:ForceUpdate( "SPEC_PACKAGE_CHANGED" )
                 order = 3.2,
             },
 
+            dotCap = {
+                type = "range",
+                name = "DoT Cap",
+                desc = "If set above zero, this ability will not be recommended if there are this many or more active DoTs from this ability, unless the DoT is refreshable on your main target.\n\n" ..
```

Comment:
`...not be recommended when it has been applied to this number of targets (or more). If the aura is refreshable on your current target, this limit is ignored.`